### DISCUSSION TITLE
Any with vector graphic experience

### DISCUSSION THREAD

**Discussion by WaldemarH** [OpenSalamander/salamander#58] [General]
Hi guys

does anybody have vector graphic experience. I'm creating svg icons but as I have limited knowledge about CorelDraw it's hard for me to make icons.

Any help would be welcome... if there is here nobody then we can ask on the forum if there is someone who has the knowledge.

BrW

**Comment by melloware** [comment]
@WaldemarH have you considered just grabbing Vector Icons from FontAwesome or one of the other SVG vector icon libs out there instead of creating your own?

**Reply by WaldemarH** [reply]
I have the issue is that they are too thin... the icon on the left is exact mirror of Awesome font icon.
.. download and expand video to see anything
https://drive.google.com/file/d/1--E09cF7Rr3IQtbxW-kZnQn5jLDPxi4h/view?usp=sharing

So the icons have to be fatten, but not too much or they just become a blob.

Now as with toolbar icons (which are really cool), the svg icons will now be open to change (look in the toolbars folder in AS installed folder). So users will be able to change them however they will like it.

Anyway I'll just leave it as they are and when the new version will be released we'll ask some vector graphic person to pimp them.

Maybe Jan can share who made the toolbar icons and maybe we can get some help there?

BrW

**Comment by WaldemarH** [comment]
It's unbelivable what a good tutorial does... I'm almost a master now. ;) ... good enough to create simple icons. :)

**Comment by WaldemarH** [comment]
So icons should be good now.

![icons](https://github.com/user-attachments/assets/65d8cca8-384f-42f2-806b-43e011e0530f)

**Reply by melloware** [reply]
Nice!

**Comment by benabraham** [comment]
Ping me if you need any help. I'm a web developer/designer with 25 years of experience.
